game_concepts_l_english.yml

```diff
@@ -1210,99 +1210,99 @@ game_concept_electors:0 "Eleitores"
-game_concept_lover:0 "Lover"
-game_concept_lovers:0 "Lovers"
-game_concept_lover_possessive:0 "Lover's"
-game_concept_lovers_possessive:0 "Lovers'"
+game_concept_title_succession:0 "Sucessão de Títulos"
+game_concept_title_specific_succession:0 "Específico ao Título"
+game_concept_title_succession_desc:0 "Este [title|E] tem sua própria [succession_law|E], diferente do [realm|E] deste Proprietário."
+game_concept_designated_heir:1 "Herdeiro Designado"
+game_concept_designated_heir_desc:3 "O $game_concept_designated_heir$ tem prioridade na decisão de quem se torna o [realm_heir|E]."
+game_concept_incest:0 "Incesto"
+game_concept_incestuous:0 "Incestuoso"
+game_concept_incest_desc:0 "Incesto se refere à atividade sexual entre parentes próximos.\nQuais relações são incestuosas são definidas pela $doctrine_consanguinity_name$ [doctrine|E] de [faith|E] de uma personagem."
+game_concept_relation:0 "Relação Especial"
+game_concept_relations:0 "Relações Especiais"
+game_concept_relation_desc:3 "$game_concept_relations$ são relações que uma personagem tem que não são relacionadas a família ou [vassalage|E]. Estas $game_concept_relations$ se encaixam em diferentes categorias, podem trazer [character_modifiers|E] para a personagem envolvida, e determina o que pode acontecer entre elas.\n\nRelações Padrão:\n$EFFECT_LIST_BULLET$ [friend|E]\n$EFFECT_LIST_BULLET$ [lover|E]\n$EFFECT_LIST_BULLET$ [rival|E]\n$EFFECT_LIST_BULLET$ [guardian|E]\n$EFFECT_LIST_BULLET$ [ward|E]\n$EFFECT_LIST_BULLET$ [mentor|E]\n$EFFECT_LIST_BULLET$ [student|E]\n\nUpgraded Relations:\n$EFFECT_LIST_BULLET$ [best_friend|E]\n$EFFECT_LIST_BULLET$ [soulmate|E]\n$EFFECT_LIST_BULLET$ [nemesis|E]\n\nChildhood Relations:\n$EFFECT_LIST_BULLET$ [bully|E]\n$EFFECT_LIST_BULLET$ [victim|E]\n$EFFECT_LIST_BULLET$ [crush|E]"
+game_concept_lover:0 "Amante"
+game_concept_lovers:0 "Amantes"
+game_concept_lover_possessive:0 "Do Amante"
+game_concept_lovers_possessive:0 "Dos Amantes"
 game_concept_lover_desc:3 "A $game_concept_lover$ is a $game_concept_relation$ indicating an ongoing amorous and sexual association between two [characters|E].\n\nAs they often take place outside the bonds of [marriage|E] or [concubinage|E], most $game_concept_lover$ $game_concept_relations$ begin as [secrets|E].\n\nIf a man and a woman are $game_concept_lovers$, the woman may become pregnant with the man's child. If they are not $game_concept_married$ (and one is not the $game_concept_concubine$ of the other), the child will be a [bastard|E], secret or otherwise.\n\n$game_concept_lovers$ who are deeply loyal and affectionate might become [soulmates|E]."

```

Comment:
game_concept_lover_desc:3 "Um $game_concept_lover$ é uma $game_concept_relation$ que indica uma associação amorosa e sexual em andamento entre duas [characters|E].\n\nComo normalmente elas ocorrem fora do contrato de [marriage|E] ou [concubinage|E], a maioria das $game_concept_relations$ de $game_concept_lovers$ começam como  [secrets|E].\n\nSe um homem e uma mulher são $game_concept_lovers$, a mulher pode engravidar deste homem. Se eles não são $game_concept_married$ (e um deles não é $game_concept_concubine$ do outro), a criança será um [bastard|E], se isso for um segredo ou não.\n\n$game_concept_lovers$ que são profundamente leais e afetuosos podem se tornar [soulmates|E]."